### PR TITLE
Add infobox for new developers.

### PR DESCRIPTION
**Pull request by jagtalon**
@MariagraziaAlastra what do you think? I think I need help finalizing the text:

screen shot 2015-12-10 at 3 25 12 am

Notes:
- Once it's been dismissed, it won't show up again until the "?" button is clicked.
- It shows up for everyone and not only new contributors.

https://v.usetapes.com/Mc3jhAW7Ik

### CODE REVIEW

File: src/js/ia/IAPage.js [hunk 1]
```diff
@@ -155,6 +155,25 @@
                         $(".ia-issues ul li").show();
                     });
 
+		    if(localStorage.dismissInfo) {
```
Comment (MariagraziaAlastra) [inline src/js/ia/IAPage.js]: @jagtalon I'm curious why did you choose to use localStorage instead of checking the new_contributor value?

**Comment by jagtalon** [comment]
@MariagraziaAlastra ok ready for review again

**Comment by MariagraziaAlastra** [comment]
It's critical to get the wording right here before merging.

File: templates/instantanswer/ia.tx [hunk 2]
```diff
@@ -42,61 +42,47 @@
...
+	  <h1 class="pull-left tx-size--medium tx-clr--dk">Next Steps</h1>
+	  <span class="ddgsi-close-bold tx-size--medium pull-right"></span>
+	</header>
+	<div class="gw infobox__container">
+	  <div class="g third">
+	    <h2 class="tx-size--small tx-clr--dk">
+	      <strong>Set Up Your Dev Environment</strong>
+	    </h2>
+	    <p>
+	      <a href="https://duck.co/duckduckhack/setup_dev_environment"> Here </a> we'll give a step-by-step guide on how to get your development environment ready.
```
Comment (russellholt) [inline templates/instantanswer/ia.tx]: simplify to

[Setup Guide](https://duck.co/duckduckhack/setup_dev_environment)

File: templates/instantanswer/ia.tx [hunk 3]
```diff
@@ -42,61 +42,47 @@
...
+              <a href="https://duckduckhack.slack.com/messages/ask-anything/">Slack channel</a>!
+	    </p>
+	  </div>
+	  <div class="g third">
+	    <h2 class="tx-size--small tx-clr--dk">
+	      <strong>Fork the Repo</strong>
+	    </h2>
+	    <p>
+	      After getting your environment set up, make sure you fork the right repo to start coding. Check out <a href="https://duck.co/duckduckhack/setup_dev_environment#fork-the-correct-duckduckhack-repository-on-github"> the docs </a>
+              for more information.
```
Comment (russellholt) [inline templates/instantanswer/ia.tx]: Change to link to the correct repo, eg for spice,

Fork the [zeroclickinfo-spice](link to correct repo) repo on github to get started. For more information see [Choosing the correct repo](https://duck.co/duckduckhack/setup_dev_environment#fork-the-correct-duckduckhack-repository-on-github).

File: templates/instantanswer/ia.tx [hunk 4]
```diff
@@ -42,61 +42,47 @@
...
+	    </p>
+	  </div>
+	  <div class="g third">
+	    <h2 class="tx-size--small tx-clr--dk">
+	      <strong>Create a Pull Request</strong>
+	    </h2>
+	    <p>
+	      When everything is ready, send us your code by creating a <a href="https://duck.co/duckduckhack/preparing_for_a_pull_request">pull request</a>.
```
Comment (russellholt) [inline templates/instantanswer/ia.tx]: change to

When your IA is ready for initial review, [create a pull request](https://duck.co/duckduckhack/preparing_for_a_pull_request). Be sure to paste the link to this IA page in the PR description:

IA: [title of this IA](link to this actual IA page)

**Comment by russellholt** [comment]
Text just needs to be cleaned up (made suggestions) and we need to be sure we are linking to the correct repo to fork.

**Comment by jagtalon** [comment]
Closing this. Moving to another branch.

**Pull request closed without merging**